package.json

```diff
@@ -164,11 +164,17 @@
       ],
       "order": 8
     },
+    "cleanAfterBuild": {
+      "title": "Clean after a build.",
```

Comment:
Change the title to `"Clean after Build"`, and perhaps consider adding a description giving some additional details, such as the fact that cleanup (should) only take place after a *successful* build.